kuma/demos/tests/test_helpers.py

```diff
@@ -14,9 +12,9 @@ def test_tag_description_no_description(self):
             "title": "Testing tag without description",
         }
         description = tag_description("test_tag")
-        eq_("test_tag", description)
+        self.assertEqual("test_tag", description)
```

Comment:
If pytest is the eventual goal, why not just `assert 'test_tag' == description` since pytest monkey patches assert to make it nice?